aspnetcore/signalr/dotnet-client/sample/SignalRChatClient/SignalRChatClient.csproj

```diff
@@ -34,92 +34,83 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Microsoft.AspNetCore.Connections.Abstractions, Version=2.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL">
-      <HintPath>..\packages\Microsoft.AspNetCore.Connections.Abstractions.2.1.0-rc1-final\lib\netstandard2.0\Microsoft.AspNetCore.Connections.Abstractions.dll</HintPath>
+    <Reference Include="Microsoft.AspNetCore.Connections.Abstractions, Version=2.1.3.0, Culture=neutral, PublicKeyToken=adb9793829ddae60, processorArchitecture=MSIL">
```

Comment:
I missed that this sample originally used `packages.config`-style NuGet references. @scottaddie if it's easy, would you be able to try converting it to `<PackageReference>`? If it's going to take more than the built-in VS gesture, I don't think it's worth bogging down this PR, but it would be nice to use the modern stuff.

Reply:
Ah, this is a WPF app. I *think* `<PackageReference>` should still work there, and VS will still provide the gesture. If not, let's just leave it as-is.

Reply:
Great idea! VS does provide the gesture. I saw the *packages.config* file and wanted to run away screaming.